browser/js/containers/Grade/index.js

```diff
@@ -31,12 +55,13 @@ const styles = {
   }
 }
 
-export default class GradeView extends Component {
+export default class Grade extends Component {
   constructor(props) {
     super(props)
     this.state = {
       isLoading: true,
-      content: ''
+      content: '',
+      current: 'Home'
     }
   }
 
```

Comment:
should this be an action? seems like we might want to track the current repo view across the app.